backend/src/resolvers/comments.js

```diff
@@ -53,6 +53,11 @@ export default {
       )
       session.close()
 
+      return comment
+    },
+    DeleteComment: async (object, params, context, resolveInfo) => {
+      const comment = await neo4jgraphql(object, params, context, resolveInfo, false)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-03T12:53:15Z" title="Monday, June 3rd 2019, 2:53:15 pm +02:00">Jun 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -53,6 +53,11 @@ export default {
       )
       session.close()
 
+      return comment
+    },
+    DeleteComment: async (object, params, context, resolveInfo) => {
+      const comment = await neo4jgraphql(object, params, context, resolveInfo, false)
```


@Tirokk if you're not doing anything you can also omit `DeleteComment`.